src/metaswitch/crest/tools/sstable_provisioning/README.md

```diff
@@ -63,14 +63,12 @@ To store the passwords for the subscribers (in plaintext), add the `plaintext_pa
 
 Now that you've got the parameters, run one of the following commands.
 
-    sudo ./BulkProvision homer <parameters>
```

Comment:
I think you want to keep these instructions, and the ones below - they're still useful in split-storage.